### PR TITLE
CI: check_freq=False

### PR DESCRIPTION
**Pull request by jbrockmendel**
Aimed at a test that has been periodically failing in the CI for a week or so.

### CODE REVIEW

File: pandas/_testing.py [hunk 1]
```diff
@@ -1140,6 +1141,8 @@ def assert_series_equal(
         Whether to compare internal Categorical exactly.
     check_category_order : bool, default True
         Whether to compare category order of internal Categoricals.
+    check_freq : bool, default True
```
Comment (simonjayhawkins) [inline pandas/_testing.py:1144]: move to below versionadded:: 1.0.2 for check_category_order and add versionadded:: 1.1.0

File: pandas/_testing.py [hunk 2]
```diff
@@ -1327,6 +1331,8 @@ def assert_frame_equal(
         If True, ignore the order of index & columns.
         Note: index labels must match their respective rows
         (same as in columns) - same labels must be with the same data.
+    check_freq : bool, default True
+        Whether to check the `freq` attribute on a DatetimeIndex or TimedeltaIndex.
```
Comment (simonjayhawkins) [inline pandas/_testing.py:1335]: add versionadded:: 1.1.0

**Comment by jreback** [comment]
looks ok, comments by @simonjayhawkins

**Comment by jbrockmendel** [comment]
comments addressed

**Review by jreback — APPROVED** [review]
lgtm merge into green

we should maybe make the default False for backwards compat here (pls open an issue)

**Comment by jbrockmendel** [comment]
green

**Comment by jreback** [comment]
k thanks. pls open an issue to change this to `check_freq=False`; I *think* this might break downstream, IIRC a comment by @jorisvandenbossche

**Comment by MaxWinterstein** [comment]
> k thanks. pls open an issue to change this to `check_freq=False`; I _think_ this might break downstream, IIRC a comment by @jorisvandenbossche

Jepp, this broke some code for me. Will open the mentioned issue.

**Pull request merged**